.github/workflows/ci.yml

```diff
@@ -29,7 +29,11 @@ jobs:
       - name: Install deps
         run: |
           apt update && apt install -y --no-install-recommends ca-certificates cmake build-essential clang llvm git pkg-config autoconf automake libtool libelf-dev wget libb64-dev libc-ares-dev libcurl4-openssl-dev libssl-dev libtbb-dev libjq-dev libjsoncpp-dev libgrpc++-dev protobuf-compiler-grpc libgtest-dev libprotobuf-dev liblua5.1-dev linux-headers-amd64
-  
+          (
+            mkdir third_party && cd third_party
+            ../.github/install-valijson.sh
```

Comment:
Now that you have a dedicated script for this, it might be worth moving the `mkdir` and `cd` inside the script as well. Further more, calling a script already spawns a subshell, so you no longer need the `()` around the call.

Reply:
Actually I just noticed you create and `cd` into a build directory in the script. You could change `build` to `third_party/valijson` in the script and (at least to me) it would be clearer.

Reply:
The goal here was to prevent downloading and extracting valijson sources directly at the root of the project.
I moved the "build location" management to the script. That's a two step process, but I don't see a really better way to achieve this.